### ISSUE TITLE
editUser dictionary parameter undocumented

### ISSUE THREAD

**Issue by thoastbrot** [EXASOL/exaoperation-xmlrpc#1]
using the same spec as for addUser, I'm getting a ValueError:

```xmlrpclib.Fault: <Fault -1: \"Could not edit user: <type 'exceptions.ValueError'>\">```

for very simple strings set like this:

```
cluster.editUser('tom', {
    'user_login': 'tom',
    'password': 'password',
    'user_title': 'devops',
    'user_description': 'some description',
})
```

leaving out the (otherwise not considered useful) user_login, I'm getting a empty string user login and the other values changed as requested, so I assume there's something wrong. But without documentation, I can't tell.

**Comment by aoby01** [comment]
Hi,
are there any news on this issue?

**Comment by florian-reck** [comment]
This issue will be fixed with Exasol v6.0.11.
